Probesys/bileto
# dev: Refactor preferences and profile forms

## Related issue(s)



https://github.com/Probesys/bileto/issues/551

## How to test manually



1. Use preferences and profile form
2. Verify everything works as expected

## Checklist
